nix/test-nixos-configuration.nix

```diff
@@ -35,8 +35,8 @@
     hostAddr = "0.0.0.0";
     socketPath = "/var/run/cardano-node/node.socket";
     systemdSocketActivation = false;
-    nodeConfigFile = "${cardano-configurations}/network/preview/cardano-node/config.json";
-    topology = "${cardano-configurations}/network/preview/cardano-node/topology.json";
+    nodeConfigFile = "${cardano-configurations}/network/sanchonet/cardano-node/config.json";
```

Comment:
We need to switch back to preview

Reply:
Addressed in https://github.com/Plutonomicon/cardano-transaction-lib/pull/1638/commits/5a745da036583337d658b27a4dd96dcfd5ce3699